From e70e9320b7d832c8bc54176d3f21bce7ad55f4d3 Mon Sep 17 00:00:00 2001
From: Rivenderu <Kuvampa@gmail.com>
Date: Wed, 5 Apr 2023 16:15:58 +0300
Subject: [PATCH 2/2] Update another two lines in git_1 firstFile.txt

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index 1bd9014..5d7d1e7 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -4,7 +4,7 @@
 4
 5
 6
-7
-8
+4
+3
 2
 1
-- 
2.39.1.windows.1

